Fixtures/Miscellaneous/PluginGeneratedResources/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 6.0
```

Comment:
issue (possibly-blocking): Although this seems like a very simple change, I fear this might introduce regression as the swift-tools-version for the test package was set for 5.7 for a reason.

What is the purpose of updating this?  Can we revert this back to 5.7 and maybe disable (or mark the test as expected failure)?